tuf/client_rework/updater_rework.py

```diff
@@ -35,27 +36,26 @@ class Updater:
     """
     Provides a class that can download target files securely.
 
-    Attributes:
-        metadata:
-
-        repository_name:
-
-        mirrors:
-
-        fetcher:
-
-        consistent_snapshot:
+    TODO
     """
 
     def __init__(
         self,
         repository_name: str,
-        repository_mirrors: Dict,
+        metadata_url: str,
+        default_target_url: Optional[str] = None,
         fetcher: Optional[FetcherInterface] = None,
     ):
 
         self._repository_name = repository_name
-        self._mirrors = repository_mirrors
+        self._metadata_url = metadata_url
+        # Should we accept metadata url as a default for targets or
+        # targets_url should be provided either in this constructor
+        # or as a download_target parameter?
+        if default_target_url is None:
+            self._default_target_url = metadata_url
+        else:
+            self._default_target_url = default_target_url
```

Comment:
Nit: I'd probably write this as 
```suggestion
        self._default_target_url = metadata_url
        if default_target_url is not None:
            self._default_target_url = default_target_url
```